dsp-shared/src/test/scala/dsp/valueobjects/SchemaSpec.scala

```diff
@@ -437,6 +847,21 @@ object SchemaSpec extends ZIOSpecDefault {
 
       assertTrue(guiObject.guiAttributes == scala.collection.immutable.List()) &&
       assertTrue(guiObject.guiElement == Some(guiElementCheckbox))
+    },
+    test(
+      "pass gui element 'salsah-gui#Checkbox' with gui attribute 'min=1.0' and return an error"
+    ) {
+      assertTrue(
+        Schema.GuiObject
+          .make(
+            scala.collection.immutable.List(guiAttributeMin),
+            Some(guiElementCheckbox)
+          ) == Validation.fail(
+          ValidationException(
+            SchemaErrorMessages.GuiAttributeNotEmpty
+          )
+        )
+      )
     }
   )
 }
```

Comment:
Just wonder if these tests could be simplified, because the file seems to be already long...